src/optimizer/src/rollup/index.ts

```diff
@@ -144,19 +134,14 @@ export function qwikRollup(opts: QwikPluginOptions = {}): any {
           ),
           version: '1',
         };
-
-        clientOutputEntryMaps.push(outputEntryMap);
-        writeSymbols();
-      } else if (outputOpts.format === 'cjs') {
-        if (outputOpts.dir && opts.symbolsPath) {
-          symbolsWriters.push((outputEntryMap) => {
-            this.emitFile({
-              fileName: opts.symbolsPath,
-              source: JSON.stringify(outputEntryMap, null, 2),
-              type: 'asset',
-            });
+        if (typeof opts.symbolsOutput === 'string') {
+          this.emitFile({
+            fileName: opts.symbolsOutput,
+            source: JSON.stringify(outputEntryMap, null, 2),
+            type: 'asset',
           });
-          writeSymbols();
+        } else {
```

Comment:
Maybe double check the type is a `function`

Reply:
good one